test/extended/crc/cmd/cmd_test.go

```diff
@@ -8,15 +8,17 @@ import (
 
 func TestCommand(t *testing.T) {
 	assert.Equal(t, "crc setup", Command{
-		command: "setup",
+		command:     "setup",
+		updateCheck: true,
 	}.ToString())
 	assert.Equal(t, "CRC_DEBUG_ENABLE_STOP_NTP=true crc start -b bundle", Command{
 		command:     "start -b bundle",
 		disableNTP:  true,
 		updateCheck: true,
 	}.ToString())
 	assert.Equal(t, "CRC_DISABLE_UPDATE_CHECK=true CRC_DEBUG_ENABLE_STOP_NTP=true crc start -b bundle", Command{
-		command:    "start -b bundle",
-		disableNTP: true,
+		command:     "start -b bundle",
+		disableNTP:  true,
+		updateCheck: false,
```

Comment:
could you please add some details to the commit log why this is needed now or what this fixes

Reply:
@albfan check previous comment :pray:

Reply:
I tried to be explicit about expected result from test, but updateCheck by default is false. @anjannath Should I remove this part of the change?

Reply:
ahh thanks, i understand now, so you wanted to make `Command.updateCheck = false` explicit, we can keep this change